## 502_xulrunner-2.0-window-blur.patch
##
## All lines beginning with `## GP:' are a description of the patch.
## GP: window.blur() fails on popup window in gtkmozembed. bz#325644

--- dom/src/base/nsGlobalWindow.cpp	2006-06-29 05:24:47.000000000 +0000
+++ dom/src/base/nsGlobalWindow.cpp	2006-08-09 18:55:49.000000000 +0000
@@ -3571,7 +3571,9 @@
   if (NS_SUCCEEDED(rv))
     mDocShell->SetHasFocus(PR_FALSE);
 
-  return rv;
+  // If embedding apps don't implement nsIEmbeddingSiteWindow2, we
+  // shouldn't throw to web content.
+  return NS_OK;
 }
 
 NS_IMETHODIMP
